### PR TITLE
Update ci-chat-bot deployments to work correctly for PR based jobs

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

File: ci-operator/infra/openshift/ci-chat-bot/deploy.yaml [hunk 1]
```diff
@@ -87,3 +87,5 @@ items:
           - /usr/bin/ci-chat-bot
           - --prow-config=/etc/prow/config.yaml
           - --job-config=/etc/jobs
+          - --github-endpoint=http://ghproxy
```
Comment (stevekuznetsov) [inline ci-operator/infra/openshift/ci-chat-bot/deploy.yaml]: Err what, why?

Comment (smarterclayton) [reply ci-operator/infra/openshift/ci-chat-bot/deploy.yaml]: Because the chat bot has to look at details of PRs.

Comment (smarterclayton) [reply ci-operator/infra/openshift/ci-chat-bot/deploy.yaml]: https://github.com/openshift/ci-chat-bot/blob/master/manager.go#L607

**Pull request merged**